_infra/helm/locust/locustfiles/locustfile.py

```diff
@@ -529,24 +529,34 @@ def perform_requests(self):
             grouping = request.get("grouping")
             expected_response_text = request.get("expected_response_text")
             expected_response_status = request.get("response_status", 200)
+            request_url = request["url"] if "url" in request else None
```

Comment:
request_url = request.get("url") is the same

Reply:
That's a good point.

Reply:
I reverted it back to how it originally was. It seems to be the best option.